pkg/controller/global/setup.go

```diff
@@ -14,6 +15,7 @@ type registerFunc func(context.Context, *config.Scaled, *server.Server) error
 var registerFuncs = []registerFunc{
 	settings.Register,
 	template.Register,
+	auth.Register,
```

Comment:
Why do we put it in global scope

Reply:
Because we need to update all replica's TokenManager, not only master

Reply:
Why do we need TokenManager in all replicas